union/core/bot.py

```diff
@@ -16,6 +16,9 @@ def __init__(self, config: dict):
 
         super().__init__(command_prefix=self.config.get("command_prefix", '!'))
 
+        for cog in self.config.get('cogs', []):
+            self.load_extension('union.cogs.{}'.format(cog))
```

Comment:
There should be feedback to the user when a cog is loading, if it fails to load, etc.